hushline/crypto.py

```diff
@@ -21,7 +21,7 @@ def encrypt_field(data: bytes | str | None) -> str | None:
         # If data is a string, encode it to bytes
         data = data.encode()
 
-    return fernet.encrypt(data).decode()
+    return fernet.encrypt_at_time(data, current_time=0).decode()
```

Comment:
```suggestion
    # We explicitly set the current time to 0 to avoid storing timestamps that could be used to de-anonymize user-activity per the threat model.
    return fernet.encrypt_at_time(data, current_time=0).decode()
```

Reply:
Let's add a brief comment since this may constitute abuse of Fernet in other contexts.

Reply:
Adopted by (https://github.com/scidsg/hushline/pull/466/commits/3945ac6a5e5f52ca2d03b950acf82ba8967452e8)